pkgs/applications/audio/csound/csound-qt/examplepath.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/src/qutecsound.cpp b/src/qutecsound.cpp
+index 103c006f..59340962 100644
+--- a/src/qutecsound.cpp
++++ b/src/qutecsound.cpp
+@@ -2385,7 +2385,7 @@ void CsoundQt::handleTableSyntax(QString syntax)
+ }
+ 
+ void CsoundQt::openManualExample(QString fileName)
+-{    
++{
+ #ifdef Q_OS_WIN // on windows poper path is not forwarded. Add it if necessary.
+     if (!fileName.startsWith(helpPanel->docDir)) {
+         fileName =  helpPanel->docDir + fileName;
+@@ -2947,7 +2947,7 @@ void CsoundQt::setDefaultKeyboardShortcuts()
+ 	raiseInspectorAct->setShortcut(tr("Ctrl+5"));
+ 	showLiveEventsAct->setShortcut(tr("Ctrl+6"));
+ 
+-#ifdef QCS_QTHTML	
++#ifdef QCS_QTHTML
+ 	raiseHtml5Act->setShortcut(tr("Ctrl+4")); // Alt-4 was before for Code graph
+ #endif
+ 	openDocumentationAct->setShortcut(tr("F1"));
+@@ -3971,14 +3971,17 @@ QString CsoundQt::getExamplePath(QString dir)
+ #ifdef Q_OS_LINUX
+     examplePath = QString(); //qApp->applicationDirPath() + "/Examples/" + dir;
+     QStringList possiblePaths;
++
+     possiblePaths << qApp->applicationDirPath() + "/Examples/" << "~/.local/share/csoundqt/Examples/"
+                   << "/usr/share/csoundqt/Examples/" << qApp->applicationDirPath() + "/../src/Examples/"
+                   << qApp->applicationDirPath() + "/../../csoundqt/src/Examples/"
+                   <<  "/../../qutecsound/src/Examples/"
+-                  << "~/.local/share/qutecsound/Examples/" << "/usr/share/qutecsound/Examples/";
++                  << "~/.local/share/qutecsound/Examples/" << "/usr/share/qutecsound/Examples/"
++                  << qApp->applicationDirPath() + "/../share/csoundqt/Examples/";
```

Comment:
Do you mind sending this patch upstream also?

Reply:
That's already done and merged https://github.com/CsoundQt/CsoundQt/pull/268

Reply:
Can you then please use `fetchpatch` to acquire the patch from the upstream commit:
https://github.com/CsoundQt/CsoundQt/commit/09f2d515bff638cbcacb450979d66e273a59fdec.diff

Reply:
Didn't know about fetchpatch, thanks for letting me know, also read that it's the recommended way.